src/syslog_mcp/services/alert_monitor.py

```
@@ -0,0 +1,357 @@
+                        "query": query_string,
+                        "default_field": "message"
+                    }
+                })
+            
+            # Execute search
+            response = await self.es_client._client.search(
+                index="syslog-ng",
+                body=search_query,
+                timeout="10s"
+            )
+            
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)